cli/cli.go

```diff
@@ -76,24 +77,37 @@ func Run() {
 			Name:      "list",
 			ShortName: "l",
 			Usage:     "list nodes in a cluster",
+			Flags: []cli.Flag{
```

Comment:
nit: I know it's only used here, but I prefer to have all the flags in `cli/flags.go`